src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1156,10 +1156,18 @@ private void ShowOrganelleMenu(IEnumerable<OrganelleTemplate> selectedOrganelles
 
         var count = organelles.Count;
 
+        var firstOrganelle = organelles.First();
+
         // Disable delete for nucleus or the last organelle.
         if (MicrobeSize <= count || organelles.Any(o => o.Definition == nucleus))
         {
             organelleMenu.EnableDeleteOption = false;
+
+            organelleMenu.DeleteOptionTooltip = firstOrganelle.Definition == nucleus ?
```

Comment:
Just one comment here, this logic is incorrect. Instead I think you'll need to refactor the if condition like:
```c#
        bool attemptingNucleusDelete = organelles.Any(o => o.Definition == nucleus);
        if (MicrobeSize <= count || attemptingNucleusDelete)
        {
           tooltip = attemptingNucleusDelete ? ... : ...;
```